universe/interface.go

```diff
@@ -222,6 +222,12 @@ type BaseForest interface {
 	// of assets tracked in the base Universe.
 	RootNodes(ctx context.Context) ([]BaseRoot, error)
 
+	// RegisterIssuance inserts a new minting leaf within the universe tree,
+	// stored at the base key.
+	RegisterIssuance(ctx context.Context, id Identifier, key BaseKey,
```

Comment:
I think we want to modify the naming a bit here, or just flatten it a bit. At this level of abstraction there isn't a distinction between a multiverse that contains all assets, one that contains all proofs for a given asset, and one that contains only issuance proofs for one or all assets. This is referring to the `BaseForest` interface name here, that was meant to signify the issuance-only multi-verse tree.

Reply:
I've renamed `BaseForest` to `Forest`.